data.json

```diff
@@ -52,13 +61,17 @@
   {
     "name": "new_york",
     "url": "http://gis.ny.gov/gisdata/data/ds_921/SAM_Master_Statewide.gdb.zip",
-    "hash": "9a52775995af73fe5e6cd1aeaa9b52fbe9c35389c8657dbd75f99149caa10fd9",
+    "hash": "dda07a26bb25b5c900d0d1464ade072e3aa769372bf4d9bca75fa05ddae96122",
     "file": "SAM_Master_Statewide_Database.gdb",
     "count": 4761260,
     "fields": {
-      "Address": {
-        "type": "static",
-        "value": "AddressLabel"
+      "Number": {
+        "type": "multi",
+        "value": ["PrefixAddressNumber", "AddressNumber", "SuffixAddressNumber"]
+      },
+      "Street": {
+        "type": "multi",
+        "value": ["PreModifier", "PreDirectional", "PreType", "StreetName", "PostType", "PostDirectional", "PostModifier"]
```

Comment:
Wow.  This schema matches up exactly with the usaddress parser.  They must be using the same standard (I forget what this is).

I'm doing my mapping in [`rules.yaml`](https://github.com/cfpb/grasshopper-parser/pull/33/files#diff-b18b554cc815926f6d91670d1aaca05bR125), and it's _almost_ the same order, just the first two elements are switched.  I'm currently using the order they occur in the address string, not the order of the list, but that _could_ change.  Do you know for sure that PreModifier should come before PreDirectional?  If so, I can change mine.

```yaml
    - id: &street_name_full street_name_full
      description: 'a modifier adter a street name, e.g. "Ext"'
      parts:
        - *street_name_pre_directional
        - *street_name_pre_modifier
        - *street_name_pre_type
        - *street_name
        - *street_name_post_type
        - *street_name_post_directional
        - *street_name_post_modifier
```

Huh, also looks like my `description` above is both incorrect and has a misspelling.  Guess I should fix that.